nixos/modules/services/networking/miniupnpd.nix

```diff
@@ -59,30 +59,12 @@ in
   config = mkIf cfg.enable {
     # from miniupnpd/netfilter/iptables_init.sh
```

Comment:
You can probably remove this comment (and the other one below).

Great work btw! :+1: